packages/snaps-execution-environments/package.json

```diff
@@ -135,5 +135,10 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "@metamask/superstruct": true
+    }
   }
```

Comment:
This should only be added in the root `package.json`. Running `yarn` from a package directory causes a LavaMoat error but you can ignore those. But why does `@metamask/superstruct` have scripts anyway? Are you sure it's a scripts issue instead of a policy issue?

Reply:
Ah it was a policy issue. Thanks for pointing me in the right direction!

Fixed: https://github.com/MetaMask/snaps/pull/2445/commits/291a40f7e81a7f06254efe00aa54659c68f8f218